themes/bootstrap5/templates/RecordDriver/EDS/core-fields.phtml

```diff
@@ -0,0 +1,43 @@
+<?php
+  $formatter = $this->recordDataFormatter($this->driver);
+  $coreFields = $formatter->getData($formatter->getDefaults($this->defaults ?? 'core'));
+  $dbLabel = $this->driver->getDbLabel();
+  $accessLevel = $this->driver->getAccessLevel();
+  $restrictedView = empty($accessLevel) ? false : true;
+  $searchOptions = $this->searchOptions($this->driver->getSourceIdentifier());
+?>
+<?php if ($searchOptions->showRestrictedViewWarning() && $restrictedView): ?>
+  <?=$this->render('error/restrictedViewWarning.phtml')?>
+<?php endif; ?>
+<?php if (!empty($coreFields)): ?>
+  <table <?php if (!empty($this->tableId)): ?>id="<?=$this->tableId ?>" <?php endif; ?>class="table table-striped">
+    <caption class="visually-hidden"><?=$this->transEsc($this->caption ?? 'Bibliographic Details')?></caption>
+    <?php foreach ($coreFields as $current): ?>
+      <tr class="record__biblio-value"><th><?=$this->transEsc($current['label'])?>:</th><td><?=$current['value']?></td></tr>
```

Comment:
Is $current['value'] potentially HTML-formatted, or is there a missing escape here?

We may want to revisit this template after #4004 is finalized.